modules/Hoot/ui/modals/exportData.js

```diff
@@ -98,15 +98,17 @@ export default class ExportData {
 
     getTranslationPath() {
         const selectedTranslation = this.translationSchemaCombo.node().value;
-        return this.translations.find( t => t.NAME === selectedTranslation ).PATH;
+        const translation = this.translations.find( t => t.NAME === selectedTranslation );
+        return !translation.hasOwnProperty('PATH')  ? translation.EXPORTPATH : translation.PATH;
```

Comment:
Do you recall why this had to change?

Reply:
the object OSM object w/in the object array that hoot-services returns for translations is different from the rest, it doesn't return a key 'PATH' with useful information, instead it returns 'EXPORTPATH'. I chose to make this change instead of altering the services code.

Reply:
What's the url that retrieves that?

Reply:
Found it, http://localhost:8080/hoot-services/ingest/customscript/getlist

Reply:
I think this might fix a bug that we were actually relying on for correct behavior.  Meaning the services code expects OSM tag output export requests to have no `translation` param.  I'll need to investigate before merging this.

Reply:
Ok, made some changes in https://github.com/ngageoint/hootenanny/pull/3302 that allows for use of RenderDb.js translation for both osm and ogr output formats, so this fix should be ok.